packages/atomic/src/components/search/smart-snippets/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -118,6 +136,15 @@ export class AtomicSmartSnippet implements InitializableComponent {
       getSmartSnippet: () => this.smartSnippet,
       getSnippetStyle: () => this.snippetStyle,
       getFeedbackSent: () => this.feedbackSent,
+      getSnippetMaximumHeight: this.snippetMaximumHeight
```

Comment:
I'll replace this for a `&&`

Reply:
I think I'll keep it like this, it does add 1 line here for each definition but it avoid having to deal with the case where the number is 0, thus having to make us deal with assigning `false` and everything that would entail...